src/lang.ts

```diff
@@ -10,95 +10,55 @@ function isObject(item: any): boolean {
 	return item && typeof item === 'object' && !Array.isArray(item) && !(item instanceof RegExp);
 }
 
-function copyArray(array: any[], kwArgs: CopyArgs): any[] {
+function copyArray(array: any[], inherited: boolean): any[] {
 	return array.map(function (item: any): any {
 		if (Array.isArray(item)) {
-			return copyArray(item, kwArgs);
+			return copyArray(item, inherited);
 		}
 
-		return isObject(item) ?
-			copy({
+		return !isObject(item) ?
+			item :
+			_mixin({
+				deep: true,
+				inherited: inherited,
 				sources: [ item ],
-				deep: kwArgs.deep,
-				descriptors: kwArgs.descriptors,
-				inherited: kwArgs.inherited,
-				assignPrototype: kwArgs.assignPrototype
-			}) :
-			item;
+				target: Object.create(null)
```

Comment:
I've been doing this out of habit, but I'm guessing having a blank slate isn't always going to be what other developers will want. I'll change it to `{}`.